### DISCUSSION TITLE
Tabs Based on Certain Folder Choice

### DISCUSSION THREAD

**Discussion by Kaelyn-dobbins** [squidfunk/mkdocs-material#6841] [Q&A]
Hello, I am wanting to better structure a large amount of information and wanted to see if anyone could help me with this.

From the home page of my site I would like the top level headings to change based on which option a user chooses. So if there are three links on the home page that go to different folders, can the top level headings change to the sub folders that are nested within each of those folders and only show the ones associated with the user's choice?

**Comment by alexvoss** [comment]
What you describe sounds like you want tabs as the top-level navigation and the content on the left to change depending on what tab the user selects? That is precisely what [navigation.tabs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs) does. Perhaps I misunderstood, in which case it might help if you give us some more info.

**Reply by Kaelyn-dobbins** [reply]
I would like to incorporate one more level before the content on the left builds out. So it would be similar to:
 top-level heading 1 -> top-level heading 2 (changes based on first choice) -> content on the left

**Comment by kamilkrzyskow** [comment]
From the description this looks to me just like 
- https://github.com/squidfunk/mkdocs-material/discussions/3405

Either you make some heavy customization as said in the marked answer, or become a part of the [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) sponsors and use the `projects` plugin to create projects with distinct navigation, but please be aware of the current [limitations](https://squidfunk.github.io/mkdocs-material/plugins/projects/#limitations).

**Reply by alexvoss** [reply]
I would recommend reviewing the information architecture before implementing a two level tabs logic, though. Perhaps it mixes topics, tasks and audiences and a better approach would be to design separate navigation structures along those axes? 

I have tried to show [how to create an audience-specific navigation on top of the header here](https://github.com/squidfunk/mkdocs-material/discussions/3405#discussioncomment-8560368). One advantage this has is that it requires only modification of the `header`block, which means it is unlikely to break in future versions and if it does it will be easy to fix. It would also be possible to pull out some other content into this space that makes the main navigation complicated.

You could have a look at [customizing by overriding the `tabs` block](https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks) in a similar way that I have done with the `header`block.
